apps/blackbox-exporter/release.yaml

```diff
@@ -18,20 +18,21 @@ spec:
     remediation:
       retries: 3
   values:
-    config:
-      modules:
-        http_2xx:
-          prober: http
-          timeout: 15s
-          http:
-            valid_http_versions: ["HTTP/1.1", "HTTP/2.0"]
-            follow_redirects: true
-            preferred_ip_protocol: "ip4"
-        http_401:
-          prober: http
-          timeout: 15s
-          http:
-            valid_http_versions: ["HTTP/1.1", "HTTP/2.0"]
-            valid_status_codes: [401]
-            follow_redirects: true
-            preferred_ip_protocol: "ip4"
+    pspEnabled: false
+      config:
```

Comment:
Indentation issue. The whole config node should be at the same level as pspEnabled.